package.json

```diff
@@ -24,26 +24,26 @@
-    "polkadot2:copy-assets:zh-CN": "shx mkdir -p website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current; shx cp -R docs/assets website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current",
+    "kusama:clean": "shx rm -rf node_modules/.cache; docusaurus clear kusama-guide-v2",
+    "kusama:start": "yarn mirror2; docusaurus start kusama-guide-v2",
+    "prekusama:build": "npm-run-all -ln kusama:clean -p \"fetch --isPolkadot=false\" mirror2 kusama:copy-assets --aggregate-output",
+    "kusama:build": "cross-env BUILDING=true docusaurus build kusama-guide-v2; node kusama-guide-v2/postprocess",
+    "kusama:publish-gh-pages": "cross-env PUBLISHING=true docusaurus deploy kusama-guide-v2",
+    "kusama:copy-assets": "cp -R docs/assets kusama-guide-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current",
+    "kusama:i18n:download": "yarn polkadot:crowdin:download:zh-CN",
+    "kusama:i18n:copy-translations": "cp -R website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current kusama-guide-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current",
+    "kusama:pdf": "yarn pdf --initialDocURLs \"http://localhost:3000/docs/kusama-index\" --outputPDFFilename=\"kusama-guide-v2/static/kusama-wiki.pdf\" --coverTitle=\"KUSAMA GUIDE\"",
+    "polkadot:clean": "shx rm -rf node_modules/.cache; docusaurus clear website-v2",
+    "polkadot:start": "yarn generateRegistryTable; docusaurus start website-v2",
+    "prepolkadot:build": "npm-run-all -ln polkadot:clean -p fetch generateRegistryTable mirror2 polkadot:copy-assets:* --aggregate-output",
+    "polkadot:build": "cross-env BUILDING=true docusaurus build website-v2; node website-v2/postprocess",
+    "polkadot:publish-gh-pages": "cross-env PUBLISHING=true docusaurus deploy website-v2",
+    "polkadot:pdf": "yarn pdf --initialDocURLs \"http://localhost:3000/docs/getting-started\" --outputPDFFilename=\"website-v2/static/polkadot-wiki.pdf\" --coverTitle=\"POLKADOT WIKI\"",
+    "polkadot:crowdin:upload:source": "crowdin upload sources --config ./crowdin2.yaml -b master",
+    "polkadot:crowdin:download:ru": "crowdin download --config ./crowdin2.yaml -b master -l ru",
+    "polkadot:crowdin:download:zh-CN": "crowdin download --config ./crowdin2.yaml -b master -l zh-CN",
+    "polkadot:copy-assets:ru": "shx mkdir -p website-v2/i18n/ru-RU/docusaurus-plugin-content-docs/current; shx cp -R docs/assets website-v2/i18n/ru-RU/docusaurus-plugin-content-docs/current",
+    "polkadot:copy-assets:zh-CN": "shx mkdir -p website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current; shx cp -R docs/assets website-v2/i18n/zh-CN/docusaurus-plugin-content-docs/current",
     "pdf": "npx mr-pdf --contentSelector=\"article\" --paginationSelector=\".pagination-nav__item--next > a\" --pdfFormat=\"A4\" --pdfMargin=\"90,60,80,60\"",
     "fetch": "node --experimental-json-modules scripts/fetch.mjs --isPolkadot",
     "mirror2": "node --experimental-json-modules scripts/mirror2.mjs",
```

Comment:
Please consider to change the script name too.

Reply:
Perhaps we also need to change script file names.

Reply:
Updated